react/features/base/session/functions.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import { toState } from '../redux';
+import { toURLString } from '../util';
+
+/**
+ * FIXME.
+ *
+ * @param {Function|Object} stateful - FIXME.
+ * @param {string} url - FIXME.
+ * @returns {*}
+ */
+export function getSession(stateful: Function | Object, url: string): ?Object {
+    const state = toState(stateful);
+
+    const session = state['features/base/session'].get(url);
+
+    if (!session) {
+        console.info(`SESSION NOT FOUND FOR URL: ${url}`);
+    }
+
+    return session;
+}
+
+/**
+ * FIXME.
+ *
+ * @param {Function | Object} stateful - FIXME.
+ * @returns {Object}
+ */
+export function getCurrentSession(stateful: Function | Object): ?Object {
+    const state = toState(stateful);
+    const { locationURL } = state['features/base/config'];
```

Comment:
It makes me uneasy to start relying on the location URL being stored there. We have it in 2 places: base/config and base/connection which is duplication, that *no es bueno*. I know you didn't create this, it was yours truly who did it, but as a stopgap until we had something better (such as sessions) so it worries me that we wan to rely on it even more.

Reply:
As mentioned in another comment, URL's instance is used to identify a "config session" for load config and load config failed actions. Do you have better idea on how config actions could be related with the session ? Maybe once we switch to UUID for session IDs we could use that.